index.js

```diff
@@ -6,21 +6,24 @@
 
 var Path = require('path');
 var Hapi = require('hapi');
+var vision = require('vision');
+var inert = require('inert');
 
 var handlebars = require('handlebars');
 handlebars = require('handlebars-helper-sri').register(handlebars);
 
 var helpers = require('./lib/helpers.js');
 
-var CSP_HEADER = "default-src 'none'; img-src 'self'; style-src 'self'; font-src 'self' ; frame-src 'self'"; // jshint ignore:line
-
 var server = new Hapi.Server();
+
+var CSP_HEADER = 'default-src "none"; img-src "self"; style-src "self"; font-src "self"; child-src "self"';
```

Comment:
Don't get me wrong, I'm all for compatibility.

But this is a security measure. If one doesn't keep up with the latest browsers, it's fair to say they don't want to be as secure as possible.

That's my personal opinion of course :)